nxontology/ontology.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
not sure if you were avoiding using this explicitly, but since the package is 3.7+, this should be okay

Reply:
This activates "Postponed Evaluation of Annotations" as per https://www.python.org/dev/peps/pep-0563/. Is becoming the default in Python 3.10, so seems like sooner the better!